client/src/redux/index.js

```diff
@@ -633,6 +648,38 @@ export const reducer = handleActions(
         }
       };
     },
+    [actionTypes.postWebhookToken]: state => {
+      return { ...state };
+    },
+    [actionTypes.postWebhookTokenComplete]: (state, { payload }) => {
+      const { appUsername } = state;
+      return {
+        ...state,
+        user: {
+          ...state.user,
+          [appUsername]: {
+            ...state.user[appUsername],
+            webhookToken: payload
+          }
+        }
+      };
+    },
+    [actionTypes.deleteWebhookToken]: state => {
+      return { ...state };
+    },
+    [actionTypes.deleteWebhookTokenComplete]: (state, { payload }) => {
+      const { appUsername } = state;
+      return {
+        ...state,
+        user: {
+          ...state.user,
+          [appUsername]: {
+            ...state.user[appUsername],
+            webhookToken: payload
```

Comment:
```suggestion
            webhookToken: ''
```
Or null, perhaps?  I'm not quite sure why we'd need the payload, since we know that it's been deleted.